packages/backend/src/app.ts

```diff
@@ -1,11 +1,15 @@
 import express from 'express'
 import mongoSanitize from 'express-mongo-sanitize'
+import path from 'path'
 import cors from 'cors'
-import router from './infrastructure/express/router'
 
-const app = express()
+const app: express.Application = express()
 app.use(mongoSanitize())
 
+app.use('../uploads', express.static(path.join(__dirname, 'uploads')))
```

Comment:
what is the reason to do that here? Is it used at all? Why its not in `attachment.routes.ts`?